tensorrt_llm/_torch/attention_backend/trtllm.py

```
@@ -1940,6 +1947,19 @@ def sparse_attn_predict(
         """
         raise NotImplementedError
 
+    def sparse_attn_ctx_predict(
+        self,
+        q: torch.Tensor,
```

Ruff (0.14.13)

Unused method argument: `q`

(ARG002)